### DISCUSSION TITLE
Σχετικά με το ΜΕΡΟΣ Γ της ΓΕ

### DISCUSSION THREAD

**Discussion by Katethoma** [sdy60/syllabus#26] [Q&A] [answered]
Καλησπέρα. Ήθελα να ρωτήσω πως θα μπούμε στο παιχνίδι με το bitmelo μέσω github. Εκεί που μας παρπέμπει ο σύνδεσμος από το ΜΕΡΟΣ Γ της ΓΕ , Ερώτημα 1, υπάρχουν repos από άλλους χρήστες που έχουν αναρτήσει δικές τους προσθήκες. Εμείς πως θα το συνδέσουμε με την Εργασία ?

**Comment by dimidimots** [comment]
Καλησπέρα,

Δεν νομίζω ότι χρειάζεται να χρησιμοποιήσουμε το GitHub στο συγκεκριμένο ερώτημα.

**Comment by afoscolou** [comment, marked as answer]
Θα μπεις στην πλατφόρμα της Bitmelo, θα δημιουργήσεις ένα δικό σου project και εκεί θα κάνεις τις μετατροπές που θέλεις.  Στη συνέχεια θα το κάνεις publish και το URL θα το αντιγράψεις μέσα στην εργασία.
